Doc/library/decimal.rst

```diff
@@ -621,6 +621,11 @@ Decimal objects
       Return :const:`True` if the argument is either positive or negative
       infinity and :const:`False` otherwise.
 
+   .. method:: is_integer()
+
+      Return :const:`True` if the argument is a finite integral value and
+      :const:`False` otherwise.
+
```

Comment:
Please add a `.. versionadded:: 3.8` note here and in the other relevant bits of documentation.

Reply:
Done.